templates/trust/contracts/TrustTemplate.sol

```diff
@@ -77,182 +83,182 @@ contract TrustTemplate is BaseTemplate {
         return dao;
     }
 
-    function setupDAO(string id, address[] beneficiaryKeys, address[] heirs, uint256[] heirsStake) public returns (Kernel) {
-        require(_hasDaoCache(msg.sender), ERROR_MISSING_CACHE_TOKENS_FOR_SENDER);
-        require(heirs.length == heirsStake.length, ERROR_BAD_HEIRS_LENGTH);
-        require(beneficiaryKeys.length == BENEFICIARY_KEYS_AMOUNT, ERROR_BAD_BENEFICIARY_KEYS_LENGTH);
-        uint256 blockedHeirsSupply = _calculateBlockedHeirsSupply(heirsStake);
+    function setupDAO(string _id, address[] _beneficiaryKeys, address[] _heirs, uint256[] _heirsStake) public returns (Kernel) {
```

Comment:
Nitpick: what do you think about `heirsStakes` instead, to keep the "noun" plural?